corehq/util/retry_time.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+from dimagi.utils.couch.cache.cache_core import get_redis_client
+from casexml.apps.phone.const import ASYNC_RETRY_AFTER
+
+EXPONENTIAL_RATE = 2
+
+
+class RedisExponentialBackoff(object):
+    @classmethod
+    def get_next_time(cls, event_key, base_time=None):
+        base_time = base_time or ASYNC_RETRY_AFTER
```

Comment:
I think you can set the default in the signature to be `ASYNC_RETRY_AFTER`
```py
 def get_next_time(cls, event_key, base_time=ASYNC_RETRY_AFTER):
    ... 
```

Reply:
Ah, I guess it isn't too likely that base_time will be 0 or any other falsy value